### ISSUE TITLE
In "ARTMO_script2.R" - object 'stat.join.conv' not found

### ISSUE THREAD

**Issue by dganeva** [mattia6690/ARTMO_R#1]
stat.join.id seems to have loaded the correct data:
![stat join id](https://user-images.githubusercontent.com/45239323/48855707-4c546000-edbd-11e8-96a4-a72db0971135.jpg)
, but then stops with an error:

> mmd<- join.conv$mmodel[[1]] %>% readMat %>% .$data
Error in eval(lhs, parent, parent) : object 'join.conv' not found

**Comment by mattia6690** [comment]
yes, the last pararaph in the script is just for me for testing. Atm I'm testing to extract all the variables from the Model and not only the cost function I now moved it to another personal script. So no need for that part of the code. Now I deleted it from the "script" script.

The result you have in stat.join.id already represents all the statistics for the MLA. Here you have a raw construct for analyzing the performance of each model in the db.It is not yet best represented as for the LUT but I will try to make it compatible with the plotting as soon as I can.
